lib/oli_web/live/history/revision_history.ex

```diff
@@ -360,6 +361,64 @@ defmodule OliWeb.RevisionHistory do
     end
   end
 
+  def handle_event("reset-monaco", _params, socket) do
+    {:noreply, assign(socket, details_modal_assigns: nil)}
+  end
+
+  def handle_event("edit-attribute", %{"attr-key" => attr_key}, socket) do
```

Comment:
Is there a convention for why we define some events with dashes and some with underscores?